### PR TITLE
added convenience function for automatic default lookups.

### PR DESCRIPTION
**Pull request by stschiff**
This should do it. Have a look.

### CODE REVIEW

File: R/dataprep_lookup_in_pandora.R [hunk 1]
```diff
@@ -40,3 +40,25 @@ namecol_value_from_id <- function(sidora_col_name, query_id, con, cache_dir = te
   return(res_vector)
   
 }
+
+#' convert_default_lookup
+#'
+#' A convenience function which simply transforms a given Pandora-Dataframe using all
+#' defined default lookups.
```
Comment (jfy133) [inline R/dataprep_lookup_in_pandora.R:47]:
```suggestion
#' defined default lookups. Typically will convert a pandora back-end numeric ID to a 'human readable string' actually displayed on the pandora webpage.
```

**Comment by stschiff** [comment]
OK, will give @nevrome a quick chance to look. Feel free to merge when OK.

File: R/dataprep_lookup_in_pandora.R [hunk 2]
```diff
@@ -40,3 +40,25 @@ namecol_value_from_id <- function(sidora_col_name, query_id, con, cache_dir = te
...
+#' 
+#' @param df data.frame. A Sidora/Pandora-Dataframe with sidora column names.
+#' 
+#' @return The converted dataframe with lookup-columns replaced by the actual values.
+#'
+#'
+#' @export
+convert_default_lookup <- function(df) {
+  cols2update <- names(df[sidora.core::sidora_col_name_has_aux(names(df))])
+  return (df %>%
```
Comment (nevrome) [inline R/dataprep_lookup_in_pandora.R:57]: FYI R functions do not need explicit return statements. The function returns the output of the last expression.

File: NAMESPACE [hunk 3]
```diff
@@ -5,6 +5,7 @@ export(add_prefix_to_colnames)
 export(col_name_has_aux)
 export(col_name_to_name)
 export(col_name_to_sidora_col_name)
+export(convert_default_lookup)
```
[the remark this answers is not in the document]
Comment (stschiff) [reply NAMESPACE:8]: Right. And I assume you have done that and committed, correct?

File: R/dataprep_lookup_in_pandora.R [hunk 4]
```diff
@@ -40,3 +40,25 @@ namecol_value_from_id <- function(sidora_col_name, query_id, con, cache_dir = te
   return(res_vector)
   
 }
+
+#' convert_default_lookup
+#'
+#' A convenience function which simply transforms a given Pandora-Dataframe using all
+#' defined default lookups. Typically will convert a pandora back-end numeric ID to a 'human readable string' actually displayed on the pandora webpage.
+#' 
+#' @param df data.frame. A Sidora/Pandora-Dataframe with sidora column names.
```
[the remark this answers is not in the document]
Comment (stschiff) [reply R/dataprep_lookup_in_pandora.R:49]: Ah I see. Not a bad an idea, but a bit beyond my R skills.

File: R/dataprep_lookup_in_pandora.R (hunk 2 above)
Comment (stschiff) [reply R/dataprep_lookup_in_pandora.R:57]: Ah right. OK, good to know.

File: NAMESPACE (hunk 3 above)
[the remark this answers is not in the document]
Comment (nevrome) [reply NAMESPACE:8]: Yes.

File: R/dataprep_lookup_in_pandora.R (hunk 4 above)
[the remark this answers is not in the document]
Comment (nevrome) [reply R/dataprep_lookup_in_pandora.R:49]: I'll create an enhancement issue for the far away future.

**Comment by stschiff** [comment]
Puh, please check, I'm rusty with R packages I realise. Multiple commits, I think I got everything now.

File: NAMESPACE (hunk 3 above)
[the remark this answers is not in the document]
Comment (stschiff) [reply NAMESPACE]: OK, should be done now, also with the new name.

**Comment by nevrome** [comment]
Ah - sorry. I only now ran the package checks with `devtools::check()` and saw that you forgot to indicate some namespaces explicitly. All functions (except base R) in a package have to be called with the respective namespace. So it should be `dplyr::across()` and `tidyselect::all_of()`. 

I will set up a CI/CD pipeline to catch these issues automatically in the future.

**Comment by nevrome** [comment]
Ha! We already have that, but it's only a NOTE. Ok. I will try to make the checks a bit more aggressive.

**Comment by stschiff** [comment]
OK, I've added the `tidyselect::` namespace, we already had `dplyr::across`. I ran `devtools::check()` and I don't know what else to do. Some tests fail on my end, but that seems to have to do with Pandora access and credentials.

**Comment by stschiff** [comment]
`devtools::check()` tells me `W  checking dependencies in R code (705ms)
   '::' or ':::' import not declared from: ‘tidyselect’`

I don't get it.

**Comment by nevrome** [comment]
Ja - please pull again. Before I merged the master back in with 057ac71 tidyselect was missing in the DESCRIPTION.

**Comment by nevrome** [comment]
Should be resolved now.

**Pull request merged**